lib/teracy-dev/logging.rb

```diff
@@ -60,11 +60,13 @@ def self.configure_logger_for(classname)
             log = TeracyDev::Common.yellow(log)
           when "INFO"
             log = TeracyDev::Common.green(log)
+          else
+            # display log message here with tracing when log level is debug only
```

Comment:
should inside `when "DEBUG"` for more specific

Reply:
i've updated, please check again!
```suggestion
            when "DEBUG"
```

Reply:
@datphan